stablehlo/tests/ops_stablehlo.mlir

```diff
@@ -488,6 +488,54 @@ func.func @broadcast_in_dim_unranked_operand(%arg0 : tensor<*xf32>) -> tensor<2x
 
 // -----
 
+// CHECK-LABEL: if 
+func.func @if(%pred : tensor<i1>, %branch_operand : tensor<2xf32>, %wrong_type : tensor<2xf32>) {
+  %0 = "stablehlo.if"(%pred) ({
+      "stablehlo.return"(%wrong_type) : (tensor<2xf32>) -> ()
```

Comment:
What's wrong about this type?

Reply:
Ah I think this is old exsiting naming, which I changed some names but not all. Fixed.